bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/EnOceanBindingConstants.java

```diff
@@ -173,6 +174,8 @@
                             new ChannelTypeUID(BINDING_ID, CHANNEL_OCCUPANCY), CoreItemFactory.SWITCH));
                     put(CHANNEL_MOTIONDETECTION, new EnOceanChannelDescription(
                             new ChannelTypeUID(BINDING_ID, CHANNEL_MOTIONDETECTION), CoreItemFactory.SWITCH));
+                    put(CHANNEL_VIBRATION, new EnOceanChannelDescription(
+                            new ChannelTypeUID(BINDING_ID, CHANNEL_VIBRATION), CoreItemFactory.SWITCH, "Vibration Alarm", false, true));
```

Comment:
You defined the Channel_Vibration as a state channel furthermore you are just handling ```convertToStateImpl``` in EEP A5_14_0A. So I would suggest to use the standard ctor for EnOceanChannelDescription and omit the last two parameters (false, true)

Reply:
hoping that the label comes through the meta data.